### DISCUSSION TITLE
Visual cue for Template Permissions

### DISCUSSION THREAD

**Discussion by creativearc-paul** [ExpressionEngine/ExpressionEngine#2905] [Feature Requests]
I have a site with dozens of templates. Some templates are internal for logged-in users, with varying levels of confidentiality needed. I'm not aware of a grid or list view of template permissions. Something in the template overview listing would be helpful. Perhaps just an icon to indicate 'custom permissions' compared to default, wide-open permissions.

Hunting down the template permissions through the interface is challenging if you want to find several templates that you want to limit to super admin or other member groups.

Image attached. (Toggles probably aren't the best UX, but just some sort of hint that a template has customized permissions
<img width="1180" alt="perms-templates" src="https://user-images.githubusercontent.com/224037/163223807-5cdeb9dd-60f5-4ce6-b1c1-05252b5bcb0d.png">
)

**Comment by intoeetive** [comment]
@ajohnson FYI
